client/my-sites/checkout/composite-checkout/lib/get-plan-features.ts

```diff
@@ -67,11 +68,10 @@ export default function getPlanFeatures(
 	if ( isWpComBusinessPlan( productSlug ) || isWpComProPlan( productSlug ) ) {
 		return [
 			! isMonthlyPlan && freeOneYearDomain,
-			! isMonthlyPlan && liveChatSupport,
-			String( translate( 'Install custom plugins and themes' ) ),
-			String( translate( 'Drive traffic to your site with our advanced SEO tools' ) ),
-			String( translate( 'Track your stats with Google Analytics' ) ),
-			String( translate( 'Real-time backups and activity logs' ) ),
+			String( translate( 'Install Custom Plugins and themes' ) ),
+			String( translate( 'Sell products with WooCommerce' ) ),
+			String( translate( '50GB of Storage' ) ),
+			String( translate( 'Free professional e-mail for 3 months' ) ),
 		].filter( isValueTruthy );
```

Comment:
Also, when I have the annual Business plan or Pro plan in cart, I don't see live chat listed in the features, only email support is listed. 
<img width="335" alt="Screenshot 2022-06-01 at 3 36 16 PM" src="https://user-images.githubusercontent.com/1269602/171396618-71129d83-f881-4d86-898c-ebbcaa080666.png">